v3/tutorials/07-add-a-pallet/index.mdx

```diff
@@ -93,41 +93,41 @@ For an introduction to editing `Cargo.toml` files, see the [Cargo reference docu
 
 1. Add the `pallet-nicks` crate to the list of `features` to be included in the `std` feature set by adding the following line `features` section.
 
-  ```TOML
-  [features]
-  default = ['std']
-  std = [
-    ...
-    'pallet-aura/std',
-    'pallet-balances/std',
-    'pallet-nicks/std',
-    ...
-  ]
-  ```
-
-  If you forget to update the `features` section, you might see errors similar to the following when you
-  build the native binaries:
-
-  ```
-  error[E0425]: cannot find function `memory_teardown` in module `sandbox`
-    --> ~/.cargo/git/checkouts/substrate-7e08433d4c370a21/83a6f1a/primitives/sandbox/src/../without_std.rs:53:12
-    |
-  53 |         sandbox::memory_teardown(self.memory_idx);
-    |                  ^^^^^^^^^^^^^^^ not found in `sandbox`
-
-  error[E0425]: cannot find function `memory_new` in module `sandbox`
-    --> ~/.cargo/git/checkouts/substrate-7e08433d4c370a21/83a6f1a/primitives/sandbox/src/../without_std.rs:72:18
-    |
-  72 |         match sandbox::memory_new(initial, maximum) {
-    |
+```TOML
+[features]
+default = ['std']
+std = [
+  ...
+  'pallet-aura/std',
+  'pallet-balances/std',
+  'pallet-nicks/std',
   ...
-  ```
+]
+```
+
+If you forget to update the `features` section, you might see errors similar to the following when you
+build the native binaries:
+
+```
+error[E0425]: cannot find function `memory_teardown` in module `sandbox`
+  --> ~/.cargo/git/checkouts/substrate-7e08433d4c370a21/83a6f1a/primitives/sandbox/src/../without_std.rs:53:12
+  |
+53 |         sandbox::memory_teardown(self.memory_idx);
+  |                  ^^^^^^^^^^^^^^^ not found in `sandbox`
+
+error[E0425]: cannot find function `memory_new` in module `sandbox`
+  --> ~/.cargo/git/checkouts/substrate-7e08433d4c370a21/83a6f1a/primitives/sandbox/src/../without_std.rs:72:18
+  |
+72 |         match sandbox::memory_new(initial, maximum) {
+  |
+...
+```
 
 1. Check that the new dependencies resolve correctly by running the following command:
 
-  ```bash
-  cargo check -p node-template-runtime
-  ```
+```bash
+cargo check -p node-template-runtime
+```
```

Comment:
Same concern with indentation...am I missing some new formatting convention?

Reply:
The steps and code below looks like it will render correctly, so I don't know why that above was "corrected" incorrectly (possibly)